temporalio/contrib/openai_agents/_invoke_model_activity.py

```diff
@@ -60,7 +60,21 @@ class FunctionToolInput:
     strict_json_schema: bool = True
 
 
-ToolInput = Union[FunctionToolInput, FileSearchTool, WebSearchTool]
+@dataclass
+class HostedMCPToolInput:
+    """Data conversion friendly representation of a HostedMCPTool."""
```

Comment:
Could we make these comments a bit clearer --

```python
class HandoffInput:
    """Data conversion friendly representation of a Handoff."""
@dataclass
class FunctionToolInput:
    """Data conversion friendly representation of a FunctionTool."""
@dataclass
class HostedMCPToolInput:
    """Data conversion friendly representation of a HostedMCPTool."""
```

why is `FooInput` being compared to `Foo`? And what does it mean to be data-conversion friendly?